synapse/util/retryutils.py

```diff
@@ -82,10 +82,10 @@ async def get_retry_limiter(destination, clock, store, ignore_backoff=False, **k
     retry_timings = await store.get_destination_retry_timings(destination)
 
     if retry_timings:
-        failure_ts = retry_timings["failure_ts"]
+        failure_ts = retry_timings.failure_ts
         retry_last_ts, retry_interval = (
-            retry_timings["retry_last_ts"],
-            retry_timings["retry_interval"],
+            retry_timings.retry_last_ts,
+            retry_timings.retry_interval,
         )
```

Comment:
I know this code was here before, but it seems simpler to write it as:

```python
retry_last_ts = retry_timings.retry_last_ts
retry_interval = retry_timings.retry_interval
```

I guess this is trying to match the code from above, but it seems more confusing to me.